common/src/main/resources/tachyon-default.properties

```diff
@@ -103,3 +103,6 @@ tachyon.user.localread.enable=true
 tachyon.user.localwrite.enable=true
 tachyon.user.network.netty.worker.threads=0
 tachyon.user.network.netty.timeout.ms=3000
+tachyon.user.remote.block.worker.threads=10
```

Comment:
the local one is called `tachyon.user.local.block.worker.client.threads`
but the remote one is called `tachyon.user.remote.block.worker.threads` without `client`,
is it by purpose?